releases/versioning.md

```diff
@@ -0,0 +1,75 @@
+---
+title: TiDB Versioning
+summary: Learn the version numbering system of TiDB.
+---
+
+# TiDB Versioning
+
+<Important>
+It is recommended to always upgrade to the latest patch release of your release series.
+</Important>
+
+Starting from TiDB v6.0.0, TiDB has introduced a new release series, DMR. Currently TiDB has two release series:
+
+* Long-Term Support Releases (LTS)
+* Development Milestone Releases (DMR)
+
+For the versioning system of TiDB v5.0.0 and earlier versions, refer to [Historical versioning](#historical-versioning-deprecated).
```

Comment:
This description doesn't cover v5.1-5.4 & 5.0.x. It would be better to make this clear.